build/dev-env.sh

```diff
@@ -47,21 +47,21 @@ fi
 
 HELM_VERSION=$(helm version 2>&1 | grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+') || true
 if [[ ${HELM_VERSION} < "v3.0.0" ]]; then
-  echo "Please upgrade helm to v3.0.0 or higher"
+  echo "Please upgrade helm to v3.9.0 or higher"
   exit 1
 fi
 
 KUBE_CLIENT_VERSION=$(kubectl version --client --short 2>/dev/null | grep Client | awk '{print $3}' | cut -d. -f2) || true
 if [[ ${KUBE_CLIENT_VERSION} -lt 14 ]]; then
-  echo "Please update kubectl to 1.15 or higher"
+  echo "Please update kubectl to 1.24.2 or higher"
   exit 1
 fi
 
 echo "[dev-env] building image"
 make build image
 docker tag "${REGISTRY}/controller:${TAG}" "${DEV_IMAGE}"
 
-export K8S_VERSION=${K8S_VERSION:-v1.21.12@sha256:f316b33dd88f8196379f38feb80545ef3ed44d9197dca1bfd48bcb1583210207}
+export K8S_VERSION=${K8S_VERSION:-v1.24.2}
```

Comment:
I hope we can pin to specific image sha

(IE like kindest/node:v1.24.0@sha256:0866296e693efe1fed79d5e6c7af8df71fc73ae45e3679af05342239cdc5bc8e including the digest

Reply:
I don't know how to ge the sha of kubernetes. Please help. I saw this https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md but it does not help

Reply:
Pinned to sha as I was confused about K8S sha but this is kindest/node sha

Reply:
yes, it's kindest/node's sha

Reply:
I added the sha. Please check